CONTRIBUTING.md

```diff
@@ -1,9 +1,9 @@
 # Contributing
 
-We love every form of contribution. By participating to this project, you
+We love every form of contribution. By participating in this project, you
 agree to abide to the `Airy` [code of conduct](/code_of_conduct.md).
 
-Please read to our [contribuing guide](/docs/docs/contributing.md) to
+Please read to our [contributing guide](/docs/docs/guidelines/contributing.md) to
```

Comment:
```suggestion
Please read our [contributing guide](/docs/docs/guidelines/contributing.md) to
```

Reply:
Wow, so I found two issues in that sentence, but missed the third :D Thank you!